code/src/components/Product.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+
+export const Product = () => {
+  const scan = useSelector((state) => state.products.product)
+  let formattedIngredientAnalysis = []
+  let formattedNutritients = []
+  let formattedAllergens = []
+  console.log(scan)
+
+  if(!scan) return null
+
+  if(scan.product && scan.product.ingredients_analysis_tags) {
+    formattedIngredientAnalysis = scan.product.ingredients_analysis_tags.map(ingredient => 
+      ingredient.replace(/\w+:/, ' ').replace(/-/gi,' ').replace(':', ' ')
+    )
+  }
+
+  if(scan.product && scan.product.nutrient_levels_tags) {
+    formattedNutritients = scan.product.nutrient_levels_tags.map(ingredient => 
+      ingredient.replace(/\w+:/, ' ').replace(/-/gi,' ').replace(':', ' ')
+    )
+  }
+
+  if(scan.product && scan.product.allergens_tags) {
+    formattedAllergens = scan.product.allergens_tags.map(ingredient => 
+      ingredient.replace(/\w+:/, ' ').replace(/-/gi,' ').replace(':', ' ')
+    )
+  }
```

Comment:
Looks like all of these does the same kind of formatting? This would be a good time where to create a little formatter function and pass in whatever array you want in it and receive the formatted versions.
Something like:
```
const formatter = (array) => {
  const formattedArray = array.map(ingredient => 
      ingredient.replace(/\w+:/, ' ').replace(/-/gi,' ').replace(':', ' ')
    )
  return formattedArray
}
```
you could even have this stored in a helpers.js file where you can have multiple functions that does things like this and then just import that function in the files where you want them.